internal/bminventory/inventory.go

```diff
@@ -2692,6 +2713,8 @@ func filterReplyByType(params installer.PostStepReplyParams) (string, error) {
 		stepReply, err = filterReply(&models.NtpSynchronizationResponse{}, params.Reply.Output)
 	case models.StepTypeFioPerfCheck:
 		stepReply, err = filterReply(&models.FioPerfCheckResponse{}, params.Reply.Output)
+	case models.StepTypeContainerImageAvailability:
+		stepReply, err = filterReply(&models.ContainerImageAvailabilityResponse{}, params.Reply.Output)
```

Comment:
What's this for?
You do Unmarshal in processImageAvailabilityResponse.

Reply:
I'm not sure actually. I just continued that function...
@filanov Is this still in use? It seems it returns the type but we always consider it as a string.

Reply:
question to @gamli75

Reply:
In general it filter out any unexpected reply. 
So if you have a model and someone added extra data to it you will not get it.

Reply:
Still, that is already taken care of in processImageAvailabilityResponse